lib/spirv/lib/src/transpiler.dart

```diff
@@ -504,6 +547,18 @@ class _Transpiler {
     functionTypes[id] = _FunctionType(returnType, params);
   }
 
+  void opConstantTrue() {
+    final int type = readWord();
```

Comment:
type isn't used, so you can increment the read position here instead of calling a method with
```
// skip type operand
position++;
```